portals/devportal/src/main/webapp/source/src/app/components/Apis/Chat/ChatMessages.jsx

```diff
@@ -71,7 +71,7 @@ function ChatMessages(props) {
                 marketplaceAssistantDocLink: (
                     <Link
                         onClick={() => {
-                            window.location.href = 'https://apim.docs.wso2.com/en/latest/get-started/overview/';
+                            window.location.href = 'https://apim.docs.wso2.com/en/4.3.0/consume/discover-apis/marketplace-assistant/';
```

Comment:
This will open up the doc in the same browser window. Let's use a anchor tag with target="_blank" instead.